internal/converter/internal/otelcolconvert/testdata/debug.alloy

```diff
@@ -0,0 +1,17 @@
+otelcol.receiver.otlp "default" {
+	grpc { }
+
+	http { }
+
+	output {
+		metrics = [otelcol.exporter.debug.default.input]
+		logs    = [otelcol.exporter.debug.default.input]
+		traces  = [otelcol.exporter.debug.default.input]
+	}
+}
+
+otelcol.exporter.debug "default" {
+	verbosity           = "Detailed"
```

Comment:
why is the first letter uppercase? that does not work, does it?

Reply:
Good spot! It looks like the upstream collector unmarshals to `Detailed` and not `detailed`. Also, it accepts any casing. So I updated our code to also accept any casing.